src/Materials/PBR/babylon.pbrBaseMaterial.ts

```diff
@@ -1189,6 +1193,13 @@
             // Matrices
             this.bindOnlyWorldMatrix(world);
 
+            // Normal Matrix
+            if (defines.OBJECTSPACE_NORMALMAP)
+            {
+                var normalMatrix = world.toNormalMatrix();

```

Comment:
To avoid instanciation, I would recommend to get a local variable to store the normalMatrix and then add a parameter to toNormalMatrix(ref: Matrix)

Reply:
Alright thanks, this is changed in the latest version